### PR TITLE
AEA-1450 Fix dependencies in plugins

### PR DESCRIPTION
**Pull request by DavidMinarsch**
Fix project loading in manager

## Proposed changes

This PR fixes various issues noted during RC1 release and when using RC1.

## Fixes

-

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes and CI passes too
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

-

### CODE REVIEW

File: aea/manager/manager.py [hunk 1]
```diff
@@ -415,6 +413,7 @@ def add_project(
                 f"Failed to load project: {public_id} Error: {str(e)}", e
             )
 
+        self._versionless_projects_set.add(public_id.to_any())
```
Comment (DavidMinarsch) [inline aea/manager/manager.py:416]: This is what caused the pesky error @panasevychol . When the project failed to load, the second time we would not be able to add it due this being placed wrongly.

File: plugins/aea-ledger-fetchai/setup.py [hunk 2]
```diff
@@ -35,7 +35,12 @@
     license="Apache-2.0",
     description="Python package wrapping the public and private key cryptography and ledger API of Fetch.AI.",
     packages=find_packages(include=["aea_ledger_fetchai*"]),
-    install_requires=["aea>=1.0.0rc1, <2.0.0", "ecdsa>=0.15", "bech32==1.2.0"],
+    install_requires=[
+        "aea>=1.0.0rc1, <2.0.0",
+        "ecdsa>=0.15",
+        "bech32==1.2.0",
+        "pyaes>=1.6.1",
+    ],
```
Comment (DavidMinarsch) [inline plugins/aea-ledger-fetchai/setup.py:43]: @panasevychol  thanks for pointing it out!

Comment (DavidMinarsch) [reply plugins/aea-ledger-fetchai/setup.py:43]: @solarw we should extend the script which checks for dependencies. And probably also have a standalone install in a clean env so we can be sure the plugins are complete.

**Review by panasevychol — APPROVED** [review]

**Review by 5A11 — APPROVED** [review]
LGTM

**Pull request merged**